PlayerCore/components/AdFinishTracker.swift

```diff
@@ -13,7 +13,7 @@ public enum AdFinishTracker {
 func reduce(state: AdFinishTracker, action: Action) -> AdFinishTracker {
     switch action {
     
-    case is ShowMP4Ad, is ShowVPAIDAd, is ShowAd:
+    case is AdRequest:
```

Comment:
Why so?

Reply:
Because we get a new id with ad request action and some detectors, that have a set of ids, will fire an extra tracker.